proto/prysm/v1alpha1/attestation.go

```diff
@@ -320,6 +461,17 @@ func (a *AggregateAttestationAndProof) Version() int {
 	return version.Phase0
 }
 
+// IsNil --
+func (a *AggregateAttestationAndProof) IsNil() error {
+	if a == nil {
+		return errors.New("aggregateAttestationAndProof is nil")
```

Comment:
I don't think we should lowercase the name of exported types in error messages.

Reply:
you can always say `nil AggregateAttestationAndProof value` if you're trying to follow lowercase error style guide rules.